extensions/wikia/GlobalWatchlist/GlobalWatchlistTask.class.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+use Wikia\Tasks\Tasks\BaseTask;
+use \Wikia\Tasks\AsyncTaskList;
+
+class GlobalWatchlistTask extends BaseTask {
+
+	const tableName = 'global_watchlist';
+	const columnUserID = 'gwa_user_id';
+	const columnCityID = 'gwa_city_id';
+	const columnNameSpace = 'gwa_namespace';
+	const columnTitle = 'gwa_title';
+	const columnRevisionID = 'gwa_rev_id';
+	const columnTimeStamp = 'gwa_timestamp';
+	const columnRevisionTimeStamp = 'gwa_rev_timestamp';
```

Comment:
:+1:  for making these constants. Any reason we should not move these to `GlobalWatchlistBot` class so that we can use them there too?